app/gui/view/component-browser/searcher-list-panel/src/navigator.rs

```diff
@@ -2,6 +2,8 @@
 //! contains two sets of navigation buttons.
 //!
 //! See [`Navigator`] struct.
+//!
+//! See the [Component Browser Design Document](https://github.com/enso-org/design/blob/e6cffec2dd6d16688164f04a4ef0d9dff998c3e7/epics/component-browser/design.md).
```

Comment:
Two paragraphs starting with `See...` look strange. Change some, or merge it to one sentence.

Reply:
Done